docs/README.rst

```diff
@@ -299,100 +330,145 @@ This would configure the following table on BIG-IP::
+| ingress.kubernetes.io/allow-http   | boolean     | Optional  | Tells the Controller to allow HTTP traffic for HTTPS Ingress resources.             | false       |
 +------------------------------------+-------------+-----------+-------------------------------------------------------------------------------------+-------------+
-| ingress.kubernetes.io/allow-http   | boolean     | Optional  | For HTTPS Ingress resources, specifies to also allow HTTP traffic.                  | false       |
-+------------------------------------+-------------+-----------+-------------------------------------------------------------------------------------+-------------+
-| ingress.kubernetes.io/ssl-redirect | boolean     | Optional  | For HTTPS Ingress resources, specifies to redirect HTTP traffic to the HTTPS port   | true        |
-|                                    |             |           | (see below).                                                                        |             |
+| ingress.kubernetes.io/ssl-redirect | boolean     | Optional  | Tells the Controller to redirect HTTP traffic to the HTTPS port for HTTPS Ingress   | true        |
+|                                    |             |           | resources (see TLS Ingress resources, below).                                       |             |
 +------------------------------------+-------------+-----------+-------------------------------------------------------------------------------------+-------------+
 
-If the Ingress resource contains a `tls` section, the `allow-http` and `ssl-redirect` annotations provide a method of controlling HTTP traffic. In this case, the controller uses the value set in the `allow-http` annotation to enable or disable HTTP traffic. Use the `ssl-redirect` annotation to redirect all HTTP traffic to the HTTPS Virtual Server.
+.. _tls ingress:
+
+TLS Ingress resources
+`````````````````````
+
+If the Ingress resource contains a `tls` section, the `allow-http` and `ssl-redirect` annotations provide a method of controlling HTTP traffic.
+In this case, the Controller uses the value set in the `allow-http` annotation to enable or disable HTTP traffic.
+Use the `ssl-redirect` annotation to redirect all HTTP traffic to the HTTPS Virtual Server.
 
-One or more SSL profiles may exist in the Ingress resource, and must already exist on the BIG-IP. The SSL profiles referenced in the Ingress resource must use the full path used on the BIG-IP, such as `/Common/clientssl`.
+You can specify one (1) or more SSL profiles in the Ingress resource.
 
-To configure health monitors on your Ingress resource, you need to use the appropriate annotation with a JSON object containing an array of health monitor JSON object for each path specified in the Ingress resource. Each health monitor JSON object must have the following 4 fields::
+- Each profile must already exist on the BIG-IP system.
+- Provide the full path to each profile on the BIG-IP system, such as `/Common/clientssl`.
```

Comment:
It may be worth noting that the profile can also be loaded from a Kubernetes Secret. The controller will look for a Secret with the name(s) provided, and if no Secret exists, then the controller will assume that the name(s) are profiles that already exist on the BIG-IP.

Reply:
addressed